guides/msearch.md

```diff
@@ -0,0 +1,74 @@
+# Multi-Search (msearch)
+
+OpenSearch provides a powerful Multi-Search (msearch) API that enables you to perform multiple search operations for documents in an index. The `msearch` API supports various parameters that allow you to customize the search operation. In this guide, we will explore functionalities such as creating an index, adding documents, performing a multi-search, and cleaning up the index afterward. To complement your exploration of the OpenSearch Multi-Search API and its parameters, a working sample is available in [`../samples/msearch.js`](../samples/msearch.js).
+
```

Comment:
The focus should be on `msearch`, not other functionalities of OS. I would also highlight the different between `msearch` and `search` (i.e. when to use msearch over search)

Reply:
Looks great! Thanks!